src/Entity.php

```diff
@@ -3892,6 +3982,31 @@ public static function getSpecificValueToDisplay($field, $values, array $options
                     return __('No automatic transfer');
                 }
                 return Dropdown::getDropdownName('glpi_transfers', $values[$field]);
+
+            case 'is_contact_autoupdate':
+            case 'is_user_autoupdate':
+            case 'is_group_autoupdate':
+            case 'is_location_autoupdate':
+            case 'is_contact_autoclean':
+            case 'is_user_autoclean':
+            case 'is_group_autoclean':
+            case 'is_location_autoclean':
```

Comment:
For the `autolean` fields, returned value should be `__('Do not delete')` or `__('Clear')`.